File: CookingPapaAngular/src/app/Models/recipeReviewVM.ts
//For put/post RecipeReviewController
export interface RecipeReviewVM{
    recipeReviewId: number | null;
    userId: number;
    recipeId: number;
    recipeReviewRating: 0|1|2|3|4|5;
    recipeReviewComment: string;
}

File: CookingPapa/CookingPapa.Domain/ViewModels/RecipeReviewVM.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CookingPapa.Domain.ViewModels
{
    public class RecipeReviewVM
    {
        public int? RecipeReviewId { get; set; }
        public int UserId { get; set; }
        public int RecipeId { get; set; }
        public int RecipeReviewRating { get; set; }
        public string RecipeReviewComment { get; set; }
    }
}
